Automattic/wp-calypso
# Videos UI: Remove unused header and footer

## Proposed Changes

This PR removes the unused videos UI header and footer components that were unused since their creation in #58528.

## Testing Instructions

Verify removed files are not in use, Calypso builds well and there are no bundle size changes.
